### PR TITLE
Revise dir structure a bit

### PR DESCRIPTION
**Pull request by erogluorhan**
This PR does:

- Fix a broken link
- Remove `pyviz` channel from the env file
- Renames notebooks to reflect their order under each caption
- Combine our previously called CH.2 and CH.3 into a single chapter (newer CH.2) that is named "METHODS & LIBRARIES FOR UNSTRUCTURED GRID VISUALIZATION" where we can cover visualization basics such as methods, rendering techniques, and libraries.
- Combine the previous CH.4 and CH.5 into a single chapter (newer CH.3) called "UXARRAY VISUALIZATION" that'd house both topology and data variable visualization. I preserved all those notebooks under Data Variable vis. for now, but I believe those could be further combined into fewer notebooks.

Note: The term "Chapter" as I refer to as parts of the cookbook in this text is actually implemented as "Caption" in the repo.

### CODE REVIEW

**Comment by clyne** [comment]
This will be a terrific contribution when it is complete!!!

File: _toc.yml [hunk 1]
```diff
@@ -6,29 +6,24 @@ parts:
...
-      - file: notebooks/04-topology/nodes
-      - file: notebooks/04-topology/edges
-
-  - caption: Data Variable Visualization
-    chapters:
-      - file: notebooks/05-data-vars/polygons
-      - file: notebooks/05-data-vars/triangles
-      - file: notebooks/05-data-vars/point-raster
+      - file: notebooks/03-uxarray-vis/01-grid-topology
+      - file: notebooks/03-uxarray-vis/02-data-variable
```
Comment (philipc2) [inline _toc.yml:22]: Remove the term "Visualization" from the top level markdown block (#) in each of these notebooks.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 This Cookbook showcases how to visualize Unstructured Grids using UXarray paired with community tool sets such as HoloViz and Matplotlib. It was put together as part of
 Project Raijin's NCAR Summer Internships in Parallel Computational Science (SIParCS) project, which is titled "Python Data Analysis and Visualization for Unstructured Grid Data." This cookbook is currently being adapted from
-our 2023 intern Ian's notebook which can be found [here](https://github.com/NCAR/geocat-scratch/blob/ifranda/viz-notebook/SIParCS2023/Visualization_Notebook.ipynb).
+our 2023 intern Ian's notebook which can be found [here](https://github.com/NCAR/geocat-scratch/blob/main/uxarray/SIParCS2023/Visualization_Notebook.ipynb).
```
Comment (philipc2) [inline README.md:13]: Make sure to update the chapters here.

File: _toc.yml [hunk 3]
```diff
@@ -6,29 +6,24 @@ parts:
...
-      - file: notebooks/01-intro/plot-api
+      - file: notebooks/01-intro/01-unstructured-grid-overview
+      - file: notebooks/01-intro/02-data-structures
+      - file: notebooks/01-intro/03-plot-api
 
-  - caption: Methods for Unstructured Grid Visualization
+  - caption: Methods & Libraries for Unstructured Grid Visualization
     chapters:
-      - file: notebooks/02-methods/template
+      - file: notebooks/02-methods/01-data-mapping
```
Comment (philipc2) [inline _toc.yml:15]: Capitalize the term "Mapping" from the top level markdown block (#) in this notebook

**Review by philipc2 — CHANGES_REQUESTED** [review]
Looks great!! A few small suggestions

File: _toc.yml (hunk 1 above)
Comment (erogluorhan) [reply _toc.yml:22]: Done; thanks!

File: README.md (hunk 2 above)
Comment (erogluorhan) [reply README.md:13]: Great catch, thanks!

File: _toc.yml (hunk 3 above)
Comment (erogluorhan) [reply _toc.yml:15]: Done

**Pull request merged**